api/pkg/s3/routbucketdelete.go

```diff
@@ -38,7 +38,10 @@ func (s *APIService) RouteBucketDelete(request *restful.Request, response *restf
 	} else if IsQuery(request, "lifecycle") {
 		s.BucketLifecycleDelete(request, response)
 
-	} else {
+	}else if IsQuery(request, "DefaultEncryption") {
+		s.BucketSSEDelete(request, response)
+
+	}else {
 		s.BucketDelete(request, response)
```

Comment:
when will it hit this?

Reply:
on DELETE of encryption on bucket. not supported for DAITO from UI

Reply:
Why ? Also why only Delete from Dashboard special?